### PR TITLE
readme: add notes on release policy

### PR DESCRIPTION
**Pull request by vojta001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,6 +303,16 @@ in your system configuration and
...
 in your Home Manager configuration.
 
+Releases
+--------
+
+Home Manager is developed against `nixpkgs-unstable` branch, which often causes
+it to contain tweaks for changes/packages not yet released in stable NixOS.
+To avoid breaking users configurations, Home Manager is released in branches
+corresponding to NixOS releases (e. g. `release-19.09`). These branches
+get fixes, but usually not new packages. If you need a package to be backported,
```
Comment (rycee) [inline README.md]:
```suggestion
get fixes, but usually not new modules. If you need a module to be backported,
```

File: README.md [hunk 2]
```diff
@@ -303,6 +303,16 @@ in your system configuration and
 
 in your Home Manager configuration.
 
+Releases
+--------
+
+Home Manager is developed against `nixpkgs-unstable` branch, which often causes
+it to contain tweaks for changes/packages not yet released in stable NixOS.
+To avoid breaking users configurations, Home Manager is released in branches
+corresponding to NixOS releases (e. g. `release-19.09`). These branches
```
Comment (rycee) [inline README.md]:
```suggestion
corresponding to NixOS releases (e.g. `release-19.09`). These branches
```

**Comment by rycee** [comment]
I think this is a fair description and I'd be happy to merge. I just added two minor comments.

**Comment by vojta001** [comment]
Fixed

**Comment by rycee** [comment]
Thanks! Rebased to master in 9ab4e70d179796485de1d77ed7ae12ea7200f645.

**Pull request closed without merging**